lucet-wasi/Cargo.toml

```diff
@@ -1,33 +1,37 @@
 [package]
 name = "lucet-wasi"
-version = "0.1.0"
+version = "0.1.1"
 description = "Fastly's runtime for the WebAssembly System Interface (WASI)"
 homepage = "https://github.com/fastly/lucet"
 repository = "https://github.com/fastly/lucet"
 license = "Apache-2.0 WITH LLVM-exception"
 categories = ["wasm"]
-authors = ["Adam C. Foltzer <acfoltzer@fastly.com>"]
+authors = ["Lucet team <lucet@fastly.com>"]
 edition = "2018"
 
+[features]
+update-bindings = ["bindgen"]
+
 [dependencies]
 cast = "0.2"
 clap = "2.23"
 failure = "0.1"
 human-size = "0.4"
 libc = "0.2"
-lucet-runtime = { path = "../lucet-runtime" }
-lucet-runtime-internals = { path = "../lucet-runtime/lucet-runtime-internals" }
-lucet-module-data = { path = "../lucet-module-data" }
+lucet-runtime = { path = "../lucet-runtime", version = "0.1.1" }
+lucet-runtime-internals = { path = "../lucet-runtime/lucet-runtime-internals", version = "0.1.1" }
+lucet-module-data = { path = "../lucet-module-data", version = "0.1.1" }
 nix = "0.13"
 rand = "0.6"
 
 [dev-dependencies]
-lucet-wasi-sdk = { path = "../lucet-wasi-sdk" }
-lucetc = { path = "../lucetc" }
+lucet-wasi-sdk = { path = "../lucet-wasi-sdk", version = "0.1.1" }
+lucetc = { path = "../lucetc", version = "0.1.1" }
 tempfile = "3.0"
 
-[build-dependencies]
-bindgen = "0.47"
+[build-dependencies.bindgen]
+version = "0.47"
+optional = true
```

Comment:
add a brief comment here as well if you don't mind

Reply:
Done.